apps/pie-microsite/src/assets/styles/components/_contentPage.scss

```diff
@@ -25,7 +25,7 @@ $content-max-width: 80ch;
 }
 
 .c-contentPage-header > *,
-.c-contentPage-content > *:not(.c-contentPage-img, hr, .c-contentPage-notification, section),
+.c-contentPage-content > *:not(.c-contentPage-img, hr, .c-contentPage-notification, section, div),
```

Comment:
This selector seems like it could be getting a bit unwieldy (just wondering if there might be a better way to do this potentially). Just worried that this not list could grow as it's already a bit hard to comprehend

Reply:
yeah fully agree - not quite sure the best approach if we want to keep all the max-widths declared together in this file

Reply:
Maybe we could always agree that custom components on a MD page are wrapped in specific div/class and just exclude that?

Reply:
or if its only going to apply to typographic elements, we could just list those elements